api/openapi.yaml

```diff
@@ -280,6 +280,8 @@ paths:
                     items:
                       $ref: "#/components/schemas/MeterQueryRow"
                 required:
+                  - from
+                  - to
```

Comment:
I think it's worth considering `to` to be nullable. For example when someone just wants the current month. Or would we expect users to use future dates in this case?

Reply:
I think that's fair, yes. I believe grafana also translates special date ranges to actual dates on the ui:

<img width="237" alt="Screenshot 2023-09-10 at 13 40 22" src="https://github.com/openmeterio/openmeter/assets/1226384/d7a940f2-741a-4437-9644-4677a8b6c5a5">